File: Android/MeetingManagerAndroid/app/src/main/java/com/microsoft/office365/meetingmgr/Models/EmailAddress.java
/*
 * Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
 * See LICENSE in the project root for license information.
 */
package com.microsoft.office365.meetingmgr.Models;

import com.microsoft.office365.meetingmgr.Utils;

/**
 * Representation of email address
 */
public class EmailAddress {
    public String Address;
    public String Name;

    @Override
    public String toString() {
        if (!Utils.isNullOrEmpty(Name)) {
            return Name;
        }
        return Address;
    }
}

File: UWP/Meeting_Manager_UWP/MeetingManager/Models/EmailAddress.cs
//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
//See LICENSE in the project root for license information.

namespace MeetingManager.Models
{
    public class EmailAddress
    {
        public string Address { get; set; }
        public string Name { get; set; }

        public override string ToString()
        {
            if (!string.IsNullOrEmpty(Name))
            {
                return Name;
            }
            return Address;
        }
    }
}
